apps/mobile/src/screens/adoptions/components/adoptions-list.tsx

```diff
@@ -0,0 +1,38 @@
+import { AdoptionType } from '@animavita/types';
+import { Box, Heading, View } from 'native-base';
+import { FlatList } from 'react-native';
+
+import { AdoptionCard } from './adoption-card';
+
+type Adoption = AdoptionType & {
+  id: string;
+};
+
+type AdoptionsListProps = {
+  adoptions: Adoption[];
+};
+
+export const AdoptionsList = ({ adoptions }: AdoptionsListProps) => {
```

Comment:
```suggestion
export const MyPetsList = ({ adoptions }: AdoptionsListProps) => {
```

Rename the file as well to `my-pets-list.tsx`